src/app/core/admin/user-management/admin-list-users.component.ts

```diff
@@ -150,15 +166,12 @@ export class AdminListUsersComponent extends AbstractPageableDataComponent<User>
 			});
 
 		this.sortEvent$.next(this.headers.find((header: any) => header.default) as SortChange);
-
-		super.ngOnInit();
 	}
 
 	confirmDeleteUser(user: User) {
 		const id = user.userModel._id;
-		const username = user.userModel.username;
-
-		console.error('Delete User not yet implemented.');
+		this.adminUsersService.removeUser(id);
```

Comment:
@smasucci - let's subscribe to the `http.delete` observable in this method (vs. in the service), so that we can wait until the request completes before calling `this.reload()` in the `next` handler. That'll ensure that the user listing reflects the changes from the delete.

Reply:
got it!